.github/workflows/release.yml

```diff
@@ -5,141 +5,246 @@ on:
       - 'v*'
 
 jobs:
-  build-binary:
-    name: Build binary
+
+  build-package:
+    name: Build package
     runs-on: ubuntu-latest
+    container:
+      image: ubuntu:16.04
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build darwinia
-        env:
-          RUST_TOOLCHAIN: nightly-2021-04-22
+      - name: Install dependencies
         run: |
-          .maintain/setup-darwinia.sh
+          apt update -y
+          apt install --no-install-recommends -y \
+            git curl ca-certificates \
+            make gcc g++ clang-4.0
 
-      - name: Upload deploy
-        uses: actions/upload-artifact@v2
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
         with:
-          name: darwinia-artifact
-          path: deploy
+          toolchain: nightly-2021-04-22
+          target: wasm32-unknown-unknown
+          default: true
 
-      - name: Build rust docs
-        run: cargo doc --all --no-deps --release
-      - uses: actions/setup-node@v2
+      - name: Cache target
+        uses: actions/cache@v2
         with:
-          node-version: '16'
-      - name: Install vercel
-        run: npm i -g vercel
-      - name: Deploy rust docs
+          path: target
+          key: ${{ runner.os }}-target
+
+      - name: Build darwinia
         run: |
-          mv vercel.json target/doc
-          mv target/doc target/${GITHUB_REPOSITORY#*/}
-          cd target/${GITHUB_REPOSITORY#*/}
-          vercel --token ${{ secrets.VERCEL_TOKEN }} --scope itering link --confirm
-          vercel --token ${{ secrets.VERCEL_TOKEN }} --scope itering deploy --prod
-
-      - uses: 8398a7/action-slack@v3
-        if: failure()
+          cargo build --release --target x86_64-unknown-linux-gnu
+
+      - name: Collect shared
+        run: |
+          mkdir -p shared/crab-runtime
+          mkdir -p shared/darwinia-runtime
+          mv target/x86_64-unknown-linux-gnu/release/darwinia shared/
+          mv target/x86_64-unknown-linux-gnu/release/wbuild/crab-runtime/*.wasm shared/crab-runtime
+          mv target/x86_64-unknown-linux-gnu/release/wbuild/darwinia-runtime/*.wasm shared/darwinia-runtime
+
+      - name: Upload shared
+        uses: actions/upload-artifact@v2
         with:
-          status: ${{ job.status }}
-          fields: repo,author,eventName,workflow,ref,commit
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_INCOMING_WEBHOOK_URL }}
+          name: darwinia-artifact
+          path: shared
 
+      - name: Optimize cache
+        run: .maintain/optimize-cache.sh
 
-  build-wasm:
-    name: Build wasm
+  build-subwasm:
+    name: Generate subwasm info
     runs-on: ubuntu-latest
+    container:
+      # Currently, Same with srtool version v0.9.18
+      # https://github.com/paritytech/srtool/blob/v0.9.18/Dockerfile#L1
+      image: ubuntu:20.04
```

Comment:
Same, `runs-on: ubuntu-latest` currently points to Ubuntu 20.04. Can remove this.

Reply:
There I can we can remove it.

Reply:
A question, When removed this.
https://github.com/darwinia-network/darwinia/blob/fd088d17f0451fddf020dee8affe9fa8ef3cefe0/.github/workflows/release.yml#L89
There is works?

Reply:
I can't think of any reason for now that this will stop working after we change to use the native `ubuntu-latest`.